Libs/MRML/Core/vtkMRMLSegmentationDisplayNode.h

```diff
@@ -329,6 +338,9 @@ class VTK_MRML_EXPORT vtkMRMLSegmentationDisplayNode : public vtkMRMLDisplayNode
   double Opacity2DFill{0.5};
   /// 2D outline opacity for the whole segmentation
   double Opacity2DOutline{1.0};
+
+  /// Remove unused display properties when updating segment list
```

Comment:
Tell if it is enabled by default and explain that in what cases it can be useful to disable it.